FrameworkInternals/optionalModules.py

```diff
@@ -172,65 +172,64 @@ def enableModule(moduleName, tag="master", serverString=""):
 	if not _checkTagExists(moduleInfo[moduleName]["url"], tag): return
 
 	baseDirectory = os.getcwd()
-	fwInternalsDir = baseDirectory + os.path.sep + "FrameworkInternals"
-	os.chdir(fwInternalsDir)

```

Comment:
I've got doubts if removing os.chdir here is a good idea, seeing what happens in the next lines. Please clarify.